docs/ops.html

```diff
@@ -2356,6 +2356,26 @@ <h5 class="anchor-heading"><a id="kafka_streams_task_monitoring" class="anchor-l
         <td>The total number of records dropped within this task.</td>
         <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
       </tr>
+      <tr>
+        <td>active-process-ratio</td>
```

Comment:
Noticed that we seem to be missing this task-level metric as well. Might be good to take stock of all the metrics at some point and make sure they're documented clearly, especially since it's easy to miss this doc which is outside of the usual Streams docs